src/funmisc.c

```diff
@@ -1562,3 +1567,273 @@ FUNCTION(fun_benchmark)
 
   return;
 }
+
+#ifdef HAVE_LIBCURL
```

Comment:
The following 3 helper functions are copied from bsd.c - is there a better place to store them? Or are we cool with small bits of duplicate code?